cmd/kubeadm/app/cmd/join.go

```diff
@@ -102,7 +91,21 @@ func NewCmdJoin(out io.Writer) *cobra.Command {
 
 		Often times the same token is used for both parts. In this case, the
 		--token flag can be used instead of specifying each token individually.
-		`),
+		`)
+)
+
+// NewCmdJoin returns "kubeadm join" command.
+func NewCmdJoin(out io.Writer) *cobra.Command {
+	cfg := &kubeadmapiext.NodeConfiguration{}
+	api.Scheme.Default(cfg)
+
+	var skipPreFlight bool
+	var cfgPath string
+
+	cmd := &cobra.Command{
+		Use:   "join <flags> [DiscoveryTokenAPIServers]",
```

Comment:
This is not part of this PR, but might be you can fix...

The output of this line is now:
```kubeadm join <flags> [DiscoveryTokenAPIServers] [flags]```

Could you kindly remove `<flags>`?